### PR TITLE
Update usage.md

### PR DESCRIPTION
**Pull request by ksa-real**

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -280,7 +280,7 @@ This type is returned by decoders and it used to represent parts of the decoed i
 
 Each decode value has these properties:
 - A bit range in the input
-  - Can be accessed as a binary using `tobits`/`tobytes`. Use the `start` and `size` keys to postion and size.
+  - Can be accessed as a binary using `tobits`/`tobytes`. Use the `start` and `size` keys to position and size.
```
Comment (wader) [inline doc/usage.md:283]: Thanks, maybe should even say ”to access” now when i read it?

Comment (ksa-real) [reply doc/usage.md:283]: Sure, somehow this part hasn't triggered me :)

Comment (wader) [reply doc/usage.md:283]: Same for me at times :) so thanks for reading and correcting. Should really spend time more on the documenatation for fq, usually very useful to try explain things

**Comment by wader** [comment]
Thanks, i will fix the other thing i mentioned later on.

**Pull request merged**